### PR TITLE
dashboard: support show dashboard address

### LINKED ISSUE #387: Display dashboard node in "display" command
Now in `tiup cluster display XX`, the PD leader node is displayed with `|L`. We can further extend the functionality to display PD dashboard node as well.

The PD dashboard node can be accessed via PD configuration interface, see https://github.com/pingcap/pd/pull/2261/files as a reference and PD swag file as an interface.

### PR DESCRIPTION
**Pull request by baurine**
Resolve #387 

- attach "|UI" symbol for the PD node which runs dashboard in the output of display command

Screenshots:

- When dashboard and leader are different nodes:

  企业微信20200528040852

- When dashboard and leader are the same node:

  企业微信20200528041147

### CODE REVIEW

File: pkg/meta/topology.go [hunk 1]
```diff
@@ -290,18 +290,25 @@ func (s PDSpec) Status(pdList ...string) string {
 		return "ERR"
 	}
 
+	// find dashboard node
+	dashboardAddr, err := pdapi.GetDashboardAddress()
```
Comment (baurine) [inline pkg/meta/topology.go:294]: I think it is fine to continue even meet error here, so I ignore to handle the err.

File: pkg/meta/topology.go [hunk 2]
```diff
@@ -290,18 +290,25 @@ func (s PDSpec) Status(pdList ...string) string {
...
+
 	for _, member := range healths.Healths {
 		suffix := ""
+		clientURL := member.ClientUrls[0]
```
Comment (baurine) [inline pkg/meta/topology.go:298]: I see some code in the pd doesn't judge whether `len(ClientUrls) > 0`, so I suppose here the `ClientUrls` must have values, does it?

File: pkg/meta/topology.go [hunk 3]
```diff
@@ -290,18 +290,25 @@ func (s PDSpec) Status(pdList ...string) string {
 		return "ERR"
 	}
 
+	// find dashboard node
+	dashboardAddr, _ := pdapi.GetDashboardAddress()
```
Comment (baurine) [inline pkg/meta/topology.go:294]: I think it is fine to continue even meet error here, so I ignore to handle the err.

**Comment by baurine** [comment]
@breeswish PTAL, thanks!

File: cmd/cluster/command/display.go [hunk 4]
```diff
@@ -216,14 +216,25 @@ func displayClusterTopology(clusterName string, opt *operator.Options) error {
 }
 
 func formatInstanceStatus(status string) string {
-	switch strings.ToLower(status) {
-	case "up", "healthy":
+	lowercaseStatus := strings.ToLower(status)
+
+	if strings.Contains(lowercaseStatus, "healthy") {
```
Comment (breeswish) [inline cmd/cluster/command/display.go:221]: how about using http://golang.org/pkg/strings/#HasPrefix , then you don't need to exclude unhealthy.

**Pull request merged**